tests/TestCase/Console/CommandCollectionTest.php

```diff
@@ -272,7 +270,6 @@ public function testAutoDiscoverCore()
         // These have to be strings as ::class uses the local namespace.
         $this->assertSame(RoutesCommand::class, $collection->get('routes'));
         $this->assertSame(SampleShell::class, $collection->get('sample'));
-        $this->assertSame(VersionCommand::class, $collection->get('version'));
```

Comment:
Shouldn't the version command still be present?

Reply:
The test is for auto discovered commands. Auto discovery only looks under src/Command so it's not present. 

Anyway I'll move it back as getting version is not possible in standalone packages.